sources/packages/web/src/helpers/helpers/uri.ts

```diff
@@ -1,12 +1,14 @@
 import {
+  DataTableSortByOrder,
   DataTableSortOrder,
   FieldSortOrder,
   PaginationOptions,
   PaginationParams,
 } from "@/types";
 
 /**
- * helper to append pagination sort and order to the url
+ * todo: remove sortOrder: DataTableSortOrder when all primevue datatables are removed.
+ * Helper to append pagination sort and order to the url
```

Comment:
If I am not wrong, this method can be marked as `deprecated` because the `getPaginationQueryString` is supposed to do the same that `addSortOptions` and `addPaginationOptions` in a cleaner way. Does it make sense?